src/BootstrapReport/helpers.py

```diff
@@ -272,6 +272,6 @@ def plot_min_crossings(optimal_path, crossings, alpha, replicates, estimate, std
     ax.hlines(y = botbound, xmin = replicates[-1], xmax = rbound, color = 'k', lw = 2, linestyle = (1.5, (1.5, 1)))
     ax.fill_between(x_axis, lower_band, upper_band, color = plt_set['bandcolor'], label = 'Confidence band', alpha = 0.25)
     ax.legend(edgecolor = 'k', loc = 'upper left')
-    ax.text(rbound - 0.04 * (rbound - lbound), botbound + 0.07 * (topbound - botbound), plot_data, 
-                fontsize = plt_set['legend_fontsize'], verticalalignment = 'bottom', horizontalalignment='right', bbox = props)
+    ax.text(0.955, 0.055, plot_data, fontsize = plt_set['legend_fontsize'], verticalalignment = 'bottom', 
```

Comment:
@MosesStewart, in order for the box to be equal distance from the bottom and side we need to use either (0.955, 0.045) or (0.945, 0.055). I think (0.955, 0.045) is slightly preferable since it gives more space for the replicates.

Reply:
@MosesStewart, I implemented the change in https://github.com/JMSLab/BootstrapReport/pull/25/commits/7a4342aa5431996b54ed2b5f85089835cd32f1c0, resolving.